docs/data/tree-view/rich-tree-view/editing/CustomLabelInput.tsx

```diff
@@ -0,0 +1,218 @@
+import * as React from 'react';
+import { styled } from '@mui/material/styles';
+import Box from '@mui/material/Box';
+import CheckIcon from '@mui/icons-material/Check';
+import IconButton from '@mui/material/IconButton';
+import { RichTreeView } from '@mui/x-tree-view/RichTreeView';
+import CloseRoundedIcon from '@mui/icons-material/CloseRounded';
+import {
+  TreeItem2,
+  TreeItem2Label,
+  TreeItem2Props,
+} from '@mui/x-tree-view/TreeItem2';
+import {
+  UseTreeItem2LabelInputSlotProps,
+  UseTreeItem2LabelSlotProps,
+} from '@mui/x-tree-view/useTreeItem2/useTreeItem2.types';
+import { useTreeItem2 } from '@mui/x-tree-view/useTreeItem2/useTreeItem2';
```

Comment:
```suggestion
import {
  UseTreeItem2LabelInputSlotProps,
  UseTreeItem2LabelSlotProps,
  useTreeItem2,
} from '@mui/x-tree-view/useTreeItem2';
```

+ remove the import above

Reply:
Double checked all the imports in the demos. should be fine now